src/content/products/deepsparse/enterprise.mdx

```diff
@@ -123,20 +110,22 @@ deepsparse.validate_license
 
 To validate a license on a machine:
 1. If you have successfully run `deepsparse.license`, `deepsparse.validate_license` can be used to validate that the license file is in the correct location:
-   - Run the `deepsparse.validate_license` with no arguments. If the referenced license is valid, you should get the DeepSparse Enterprise Edition splash screen printed out in your terminal window.
+   - Run the `deepsparse.validate_license` with no arguments. If the referenced license is valid, the DeepSparse Enterprise Edition splash screen should display in your terminal window.
    - If the `NM_CONFIG_DIR` environment variable was set when creating the license, ensure this variable is still set to the same value.
 2. If you want to supply the `path/to/license.txt`:
-   - Run the `deepsparse.validate_license` with `path/to/license.txt` as an argument as follows:
-   - `deepsparse.validate_license --license_path path/to/license.txt`
-   - If the referenced license is valid, you should get the DeepSparse Enterprise Edition splash screen printed out in your terminal window.
+   - Run the `deepsparse.validate_license` with `path/to/license.txt` as an argument as:<br>
+   `deepsparse.validate_license --license_path path/to/license.txt`
+   - If the referenced license is valid, the DeepSparse Enterprise Edition splash screen should display in your terminal window.
 
 If you encounter issues validating your DeepSparse Enterprise Edition License, contact [license@neuralmagic.com](mailto:license@neuralmagic.com) for help.
 
-## Features
+## DeepSparse Enterprise Edition Features
+
+### DeepSparse Server
 
-### 🔌 DeepSparse Server
+The DeepSparse Server allows you to serve models and pipelines from the terminal. The server runs on top of the popular FastAPI web framework and Uvicorn web server.
```

Comment:
```suggestion
DeepSparse Server allows you to serve models and pipelines from the terminal. The server runs on top of the popular FastAPI web framework and Uvicorn web server.
```

Reply:
I added the article "The" back into the sentence construction.